tests/fuzzers/README.md

```diff
@@ -0,0 +1,16 @@
+# Fuzzing
+Librdkafka supports fuzzing by way of Libfuzzer and OSS-Fuzz. This is ongoing work.
+
+## Launching the fuzzers
+The easiest way to launch the fuzzers are to go through OSS-Fuzz. The only prerequisite to this is having Docker installed.
+
+With Docker installed, the following commands will build and run the fuzzers in this directory:
+
+```
+git clone https://github.com/google/oss-fuzz
+cd oss-fuzz
+sudo python3 infra/helper.py build_image librdkafka
```

Comment:
Why is sudo required?

Reply:
I added explanation in the README now. The reason is that `helper.py` will create and run Docker images, and I think Docker is most often installed such that it requires sudo to create/run images. However, this is not strictly required e.g. https://github.com/sindresorhus/guides/blob/master/docker-without-sudo.md

Reply:
I'm not sure that's the norm, I run docker without sudo both on OSX and Ubuntu, so I suggest removing the sudo bits above.

Reply:
Fair and sounds good - removed!